docs/server/commands.rst

```diff
@@ -563,6 +563,16 @@ counter is older than on Pootle, that is someone has translated while the file
 was offline, then it will be rejected.  Otherwise the translations in the file
 are accepted.
 
+Available options:
+
+:option:`--user`
+  .. versionadded:: 2.7.3
+
+  Import file(s) as given user. If set this option should the username of a
```

Comment:
The sentence is incomplete, there seems to be some word missing. I'd consider rewording it as simple as *The user with the provided username must exist.*